src/elements/core/datetime/date-adapter.ts

```diff
@@ -30,6 +31,12 @@ export abstract class DateAdapter<T = any> {
    */
   public abstract getDate(date: T): number;
 
+  /**
+   * Gets the date as milliseconds since epoch.
+   * @param date
+   */
+  public abstract getTime(date: T): number;
```

Comment:
time should not be part of a date adapter. is there an alternative solution?

Reply:
I removed it since i used it only in tests